code/game/gamemodes/wizard/spellbook.dm

```diff
@@ -504,7 +456,7 @@
 /datum/spellbook_entry/summon/ghosts
 	name = "Summon Ghosts"
 	desc = "Spook the crew out by making them see dead people. Be warned, ghosts are capricious and occasionally vindicative, and some will use their incredibly minor abilties to frustrate you."
-	log_name = "SGH"
+	cost = 0

```

Comment:
No idea why coiax thought it cost 0 points originally. I looked at the original pr adding it, and it clearly cost 2 points at the time.